lib/querystring.js

```diff
@@ -267,7 +304,14 @@ function addKeyVal(obj, key, value, keyEncoded, valEncoded, decode) {
   }
 }
 
-// Parse a key/val string.
+/**
+ * Parse a key/val string.
+ * @param {string} qs
+ * @param {string} sep
+ * @param {string} eq
+ * @param {{ maxKeys?: 1000; decodeURIComponent?(v: string): string; }} [options] 
```

Comment:
```suggestion
 * @param {{
 *   maxKeys?: number;
 *   decodeURIComponent?(v: string): string;
 *   }} [options]
```
Max line length -> 80 + Lint.